rakudoc_draft_3.rakudoc

```diff
@@ -359,7 +359,9 @@ And this is a L<link to the Raku Programming Language|https://raku.org>.
 
 =head2 Metadata syntax
 
-Metadata is specified using Raku option pairs. The following are examples:
+Metadata is specified using Raku option pair, where the value portion
```

Comment:
Either: Metadata is specified using a Raku option pair
Or: Metadata is specified using Raku option pairs